packages/trader/src/Modules/Trading/Components/Form/TradeParams/trade-type-tabs.tsx

```diff
@@ -28,14 +29,11 @@ const TradeTypeTabs = observer(({ className }: TTradeTypeTabs) => {
             <ButtonToggle
                 id='dt_advanced_duration_toggle'
                 buttons_arr={tab_list.filter(({ is_displayed }) => is_displayed)}
-                name={is_turbos ? 'contract_type' : 'vanilla_trade_type'}
+                name='contract_type'
```

Comment:
did we use this name 'vanilla_trade_type' for anything ?  Can you check if this breaks this component compared to production? There was this instance where we used `name` to apply css haha

Reply:
No, that didn't break anything) Trade tab tabs is working the same as it does on production. 
I removed `vanilla_trade_type` everywhere and applied the same logic as for `Turbos`, so it's safe to use `contract_type` as name.

Reply:
👍